docs/getting_started.md

```diff
@@ -83,8 +83,87 @@ let make = () => {
 }
 ```
 
-Sweet 😎! We've executed a query with our `useQuery` hook. Notice that we didn't have to write _any_ types to get 💯% type inference and type saftey on the response. We use the `graphql_ppx` module you pass to `useQuery` to ensure that you're using the data returned by your query in a fully type safe way.
+Sweet 😎! We've executed a query with our `useQuery` hook. Notice that we didn't have to write _any_ types to get 💯% type inference and type safety on the response. We use type information included in the query module you pass to `useQuery` to ensure that you're using the data returned by your query in a fully safe way.
 
-## What's Next?
+## Can I See an Example?
 
 Check out the example in `examples/2-query` to see a more involved example of using `useQuery`, in addition to `reason-urql`'s `Query` component.
+
+## Writing Your First Mutation
+
+Awesome, so we've written our first query with `reason-urql`. What about mutations?
+
+Fortunately, writing mutations in `reason-urql` is just as easy as writing queries – just use the `useMutation` hook.
+
+```reason
+open ReasonUrql;
+/* Be sure to open the Hooks module to bring necessary types into scope. */
+open Hooks;
+
+/* Create a module with your GraphQL mutation. */
+module LikeDogMutation = [%graphql
+    {|
+    mutation likeDog($key: ID!) {
+      likeDog(key: $key) {
+        likes
+        name
+        breed
+      }
+    }
+  |}
+  ];
+
+[@react.component]
+let make = (~key: string) => {
+  /* Build your request by calling .make on your mutation, passing variables as labeled arguments. */
+  let request = LikeDogMutation.make(~key, ());
+
+  /* Pass the request to useMutation. */
+  let (_, executeMutation) = useMutation(~request, ());
+
+  <button onClick=(_e) => executeMutation()>
+      "Execute the Mutation (and Reward a Good Dog)"->React.string
+  </button>
+}
+```
+
+Great – we've successfully executed a mutation to like a dog! Existing queries that reference the mutated data will be notified that data has changed, meaning you don't need to think at all about refetching your data – it's all updated for you by `reason-urql`.
```

Comment:
Oh, this is quite nice. I didn't know that it worked like this :thinking: 
How do I make sure I'm using this feature?

Reply:
It has to do with `urql`'s observable system, which is powered by [`wonka`](https://github.com/kitten/wonka). Essentially, every operation that passes through the Client can have subscribers, and this is particularly important in the case of queries that are dependent on mutation results. When a mutation is executed, `urql` will aggressively invalidate the cache for any entity that has a shared `__typename` with the mutation. This then notifies subscribers to those cache entries (really any query that requests data with the same `__typename`) that they should update: https://github.com/FormidableLabs/urql#document-caching. The first example project in this repo shows this pretty well – if you hit the button to fire `executeMutation`, the query will be notified that the cache was invalidated and pull anew from it, causing a re-render. Hope that makes sense! You don't need to do anything to opt into this, it should just happen for you.